diagnostic.md

```diff
@@ -99,7 +125,11 @@ div {
 ```
 
 <!-- your answer starts here -->
+it will have san-serif and yellow text
```

Comment:
The font family will be Cursive. The HTML has inline styling for font-family which due to specificity, would take over.